src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -478,18 +479,22 @@ function GlobeControls(camera, domElement, engine) {
         }
     };
 
-    var getPickingPosition = (function getGetPickingPositionFn() {
-        var engineGfx = engine;
-        var position;
+    const getPickingPosition = function getPickingPosition(coords) {
+        if (enableTargetHelper) {
+            pickingHelper.visible = false;
+            cameraTargetOnGlobe.visible = false;
+        }
 
-        return function getPickingPosition(coords)
-        {
-            position = engineGfx.getPickingPositionFromDepth(coords);
-            engineGfx.renderScene();
+        const position = engine.getPickingPositionFromDepth(coords);
+        engine.renderScene();
 
-            return position;
-        };
-    }());
+        if (enableTargetHelper) {
+            pickingHelper.visible = true;
+            cameraTargetOnGlobe.visible = true;
+        }
```

Comment:
if 'enableTargetHelper' is enable there's error in picking position because helpers write in depth buffer. Picking position is do every time you click on globe.  s+click right is for selection?

> The targetHelpers are the axis on the globe where you clicked and where the target is, right

yes